storageserver/src/vespa/storageserver/app/distributorprocess.cpp

```diff
@@ -32,6 +33,21 @@ DistributorProcess::shutdown()
     _node.reset();
 }
 
+namespace {
+
+uint32_t
+validate_num_distributor_stripes(uint32_t cfg_n_stripes)
```

Comment:
Consider if function should be slightly renamed here as well

Reply:
Fixed